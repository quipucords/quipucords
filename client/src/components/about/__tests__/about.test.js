import React from 'react';
import { mount } from 'enzyme';
import About from '../about';

describe('About Component', () => {
  it('should render a basic display', () => {
    const props = {
      user: { currentUser: { username: 'admin' } },
      status: {},
      shown: true,
      onClose: jest.fn()
    };

    const component = mount(<About {...props} />);

    expect(component.render()).toMatchSnapshot();
  });
});
