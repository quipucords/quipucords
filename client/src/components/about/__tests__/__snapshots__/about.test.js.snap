// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About Component should render a basic display 1`] = `
<div
  role="dialog"
>
  <div
    class="fade modal-backdrop in"
  />
  <div
    class="fade in modal"
    role="dialog"
    style="display: block; padding-right: 0px;"
    tabindex="-1"
  >
    <div
      class="modal-dialog"
    >
      <div
        class="modal-content about-modal-pf"
        role="document"
      >
        <div
          class="modal-header"
        >
          <button
            class="close"
          >
            <span
              aria-hidden="true"
              class="pficon pficon-close"
              title="Close"
            />
            <span
              class="sr-only"
            >
              Close
            </span>
          </button>
        </div>
        <div
          class="modal-body"
        >
          <h1>
            <img
              alt="Entitlements Reporting"
              src="title.svg"
            />
          </h1>
          <div
            class="product-versions-pf"
          >
            <ul
              class="list-unstyled"
            >
              <li
                class=""
              >
                <strong>
                  Version
                </strong>
                 unknown (Build: unknown)
              </li>
              <li
                class=""
              >
                <strong>
                  Username
                </strong>
                 admin
              </li>
              <li
                class=""
              >
                <strong>
                  Browser Version
                </strong>
                  
              </li>
              <li
                class=""
              >
                <strong>
                  Browser OS
                </strong>
                 
              </li>
            </ul>
          </div>
          <div
            class="trademark-pf"
          />
        </div>
        <div
          class="modal-footer"
        >
          <img
            alt="ER"
            src="logo.svg"
          />
        </div>
      </div>
    </div>
  </div>
</div>
`;
